### ISSUE TITLE
implement a defined behavior for the sync of symlinks

### ISSUE THREAD

**Issue by julianseeger** [Qabel/qabel-desktop#315]
Currently, it is undefined how qabel deals with symlinks.
If the current implementation works fine (some useful behavior that can be described as 'not dangerous and not broken') then this issue may be closed. Otherwise the behavior can be chosen that is fastest to implement and produces some useful behavior.

**Comment by julianseeger** [comment]
Current behavior: the symlink is uploaded as an empty file. It (currently) does not produce any conflicts on the original sync and shows up as a plain empty file on other syncs and in the remotefs.

I would describe this behavior as "not dangerous" but "broken"

**Comment by thechauffeur** [comment]
We already discussed this a lot at the beginning of summer (while where discussing many complex things regarding our filesystem and such) and decided on a solution based on facts. We should probably stick to that. I currently don't remember this but I know we had documented everything we discussed back then.

**Comment by audax** [comment]
Symlink as an empty file is a decent enough behavior because it does not lead to data loss and does not produce lots of garbage files, but just a single one, which doesn't hurt.

The amount of users with symlinks in their syncs should be pretty small anyway since we only support Windows.

**Comment by jmt85** [comment]
I see no reason to close this issue. 

Although we currently only support windows does not mean that it always remains.

**Comment by thechauffeur** [comment]
I agree leaving this open but it should be postponed in the backlog.

The current behavior is definitely okay for the majority of windows users.

**Comment by audax** [comment]
The reason to close it, is that we have a defined behavior that is not harmful. Improving on that should be a new ticket imo.

**Comment by thechauffeur** [comment]
The OP stated the following
> [if behavior is] 'not dangerous and not broken' then this issue may be closed

and defined the current behavior as
>  "not dangerous" but "broken"

So we should probably not close this issue without agreeing upon this. I tend to support the OP as I also think this behavior is not dangerous but broken and therefore should be changed.

Nevertheless I also think that this issue is not big enough to put it into the milestone backlog. I do think however that it should be reevaluated afterwards and therefore tend to put it into the backlog. So let's just do this and moving on :)
